### PR TITLE
Minor FAQ updates for PTFE

### PR DESCRIPTION
**Pull request by armchairlinguist**
Moved the post-201807 tool version link up; removed access from the AMI section that is not relevant.

### CODE REVIEW

File: content/source/docs/enterprise/private/faq.html.md [hunk 1]
```diff
@@ -56,7 +56,9 @@ PTFE has the ability to automatically generate a support bundle including logs a
 
 ## Managing Tool Versions
 
-Terraform Enterprise has a control panel that allows admins to manage the versions of Terraform and Packer and their download locations.
+Terraform Enterprise has a control panel that allows admins to manage the versions of Terraform and Packer and their download locations.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/private/faq.html.md]: Should we delete "and Packer" here, since it's a post-Atlas world?

Comment (maryelizbeth) [reply content/source/docs/enterprise/private/faq.html.md]: yes - +1 to striking enterprise packer from the docs.

Comment (armchairlinguist) [reply content/source/docs/enterprise/private/faq.html.md]: It's not quite a post-Atlas world for anyone who can still see this version of the tool management, so I'd like to leave that until it really is. :)

**Comment by armchairlinguist** [comment]
@nfagerlund Yes - the internal communication ports are specific to functionality added with the installer.

**Pull request merged**